### DISCUSSION TITLE
set content security policy in hubs-cloud

### DISCUSSION THREAD

**Discussion by shamsail** [mozilla/hubs#4381] [Hubs Cloud Questions] [answered]
Hi, 
I'm using emailjs-com for custom email in the hubs-cloud and it's working fine on local setup. "npm start"  "https://localhost:8080" but when I deploy my hubs-cloud updated code to the server via "npm run deploy".. code is being deployed without any error and I can also see my other changes that are also working fine on live system after deployment. 
But the issue that I'm facing is that after deployment my emailjs-com code/implementation is not working and on browser's console I can see the error of "content security policy"
Error statement: `"Content Security Policy: The page’s settings blocked the loading of a resource at https://api.emailjs.com/api/v1.0/email/send (“connect-src”)".`

![Screenshot from 2021-06-28 19-06-53](https://user-images.githubusercontent.com/30554980/123650473-106eba80-d844-11eb-9da6-e5decd450111.png)

What should I do to resolve this issue.

Currently I have tried this in "index.html" but nothing has changed on live server after deployment

```
<meta http-equiv="Content-Security-Policy" content="connect-src 'self' 'unsafe-inline' 'unsafe-eval' https://api.emailjs.com
        https://api.emailjs.com/api/v1.0/email/send;">
```
Please help me in this regard..

**Comment by rawnsley** [comment, marked as answer]
You can set CSP in the Hubs Cloud admin pages. Full domain, space separated:

![IMG_20210629_071305](https://user-images.githubusercontent.com/303516/123746467-95d88600-d8a9-11eb-8997-71da626e6399.jpg)

**Reply by shamsail** [reply]
When I want to do something, at the end , it appears that it's in admin panel.

**Reply by shamsail** [reply]
Thanks a lot to remind me. :D
